File-Formats/PDF/Convert-HTML-To-PDF/features.md

```diff
@@ -268,7 +241,7 @@ You can download a complete working sample from [GitHub](https://github.com/Sync
 
 ## JavaScript
 
-The Blink HTML converter supports enabling or disabling the JavaScript while converting HTML to PDF. Refer to the following code snippet.
+The Blink HTML converter supports enabling or disabling the JavaScript using [EnableJavaScript](https://help.syncfusion.com/cr/file-formats/Syncfusion.HtmlConverter.BlinkConverterSettings.html#Syncfusion_HtmlConverter_BlinkConverterSettings_EnableJavaScript) property while converting HTML to PDF. Refer to the following code snippet.
```

Comment:
[EnableJavaScript]() property in BlinkConverterSettings class

Reply:
Modified